### PR TITLE
Move entire sidebar content down with padding

### PR DESCRIPTION
**Pull request by bfred-it**
This is more flexible than `position: relative` on a single element

### CODE REVIEW

**Comment by sindresorhus** [comment]
Flexible how? This doesn't work correctly when viewing the dashboard in an org.

**Comment by bfred-it** [comment]
Good catch. This other commit should fix it.

_Flexible_ meaning that it still works if GH or other extensions append other boxes/content in there.

File: extension/content.css [hunk 1]
```diff
@@ -163,8 +162,8 @@ body {
 .orgpage .user-repos {
 	position: static !important;
 }
-#org_your_repos {
-	top: 0 !important;
+.orgpage + .container .dashboard-sidebar {
+	padding-top:0;
```
Comment (sindresorhus) [inline extension/content.css]: `padding-top: 0;`

Comment (bfred-it) [inline extension/content.css]: Do you want me to squash the commits?

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by bfred-it** [comment]
❤️

**Pull request merged**